conf/messages

```diff
@@ -14,9 +14,10 @@ footer.govukHelp.url = https://www.gov.uk/help
 
 # Not subscribed to CDS
 cf.not-subscribed-to-cds.detail.title=Not registered with CDS
```

Comment:
what does `cf` signify?

Reply:
I copied it over from the code we used to create the page - I can take it ou if it makes things easier.

Reply:
It's OK. This will all most definitely change anyway as we don't have a defined mechanism yet on how to build out all our pages. We can revisit later.